core/os/keyboard.cpp

```diff
@@ -245,6 +245,87 @@ static const _KeyCodeText _keycodes[] = {
 	/* clang-format on */
 };
 
+static const _KeyCodeText _keycodes_alt[] = {
+	/* clang-format off */
+	{Key::ESCAPE                ,"⎋"},
```

Comment:
I'd move the comma to be placed just after the first entry, which is similar to what we do elsewhere in the engine (e.g. in https://github.com/godotengine/godot/blob/fbc9539764c8c932b62116146a6c8a7f3f5e67f7/core/input/input_map.cpp#L337):

```suggestion
	{Key::ESCAPE,                 "⎋"},
```